### DISCUSSION TITLE
macOS default cmd+? keybind

### DISCUSSION THREAD

**Discussion by memco** [ghostty-org/ghostty#3615] [Q&A] [answered]
On macOS cmd+? is a global shortcut to open the mac help menu which by default enables you to search the other menu items for some action to take (sort of like cmd+p in a lot of editors). Ghostty has this shortcut bound to a different action which opens the Ghostty github discussion page. How can I get back the ability to use cmd+? to open the help search?

**Comment by rhodes-b** [comment]
Ghostty should not have this bound by default
in fact it doesnt even seem like you can bind `?` according to 
https://github.com/ghostty-org/ghostty/blob/d6e76858164d52cff460fedc61ddf2e560912d71/src/input/key.zig#L255-L402

nor is it bound below for all of the default keybinds
https://github.com/ghostty-org/ghostty/blob/d6e76858164d52cff460fedc61ddf2e560912d71/src/config/Config.zig#L1907-L2422

**Reply by rhodes-b** [reply]
Afaik if its bound at the os level like you are describing it wouldnt even be sent to ghostty anyways

**Reply by rhodes-b** [reply]
https://github.com/ghostty-org/ghostty/discussions/3187
shows it shouldnt take anything bound at the os level if it exists

**Comment by rrotter** [comment]
It's working to open the Help menu for me. Is it possible you have some other remapping in play here? karabiner, bettertouchtool, etc?

**Comment by memco** [comment, marked as answer]
It may only be in the beta releases in the one version I have (Version 4b3c1b03, build 8484), but it definitely has `Ghostty Help` bound to `cmd+?`: 
![Image 12-28-24 at 1 03 PM](https://github.com/user-attachments/assets/8ca1d547-38e7-4d60-a33c-9d6ac991618a)

Maybe this is relevant: [menu code seems to associate it with `?`](https://github.com/ghostty-org/ghostty/blob/2a4e5e1c8ef322fc2b68b4abc8ef16441884f011/macos/Sources/App/macOS/MainMenu.xib#L366)? Anyway it's not happening in the stable version or in the latest beta (Version 6cbd69da, build 8515) so maybe it's an isolated incident.
